### PR TITLE
FISH-5828 (Payara6) Connection Pool HealthCheck Exposed as MicroProfile Metrics

### PR DESCRIPTION
**Pull request by jGauravGupta**

### CODE REVIEW

File: docs/modules/ROOT/pages/Technical Documentation/MicroProfile/Metrics/Vendor Metrics.adoc [hunk 1]
```diff
@@ -1,6 +1,9 @@
 [[custom-vendor-metrics]]
 = Custom Vendor Metrics
```
Comment (AlanRoth) [inline docs/modules/ROOT/pages/Technical Documentation/MicroProfile/Metrics/Vendor Metrics.adoc:3]: Can you include some text in-between headings; for [reference](https://payara.atlassian.net/wiki/spaces/SUP/pages/2149515318/Payara+Documentation+Style+Guide#Headings).

**Review by AlanRoth — APPROVED** [review]

**Review by JamesHillyard — APPROVED** [review]
The level 5 headers are smaller than normal text which looks strange, but not a blocker.

**Review by JamesHillyard — CHANGES_REQUESTED** [review]
As this is an enterprise only feature - it should be moved to the enterprise section of the docs

**Comment by JamesHillyard** [comment]
This is also Payara 5 Enterprise exclusive so should point to the `master` branch instead of `payara6-master`

**Review by JamesHillyard — APPROVED** [review]
Feature was changed to be Enterprise & Community & Payara 6 - Looks good!

**Pull request merged**